queries/water-z9.pgsql

```diff
@@ -35,6 +35,24 @@ FROM
         AND way_area::bigint > 102400 -- 4px
         AND mz_way12 && !bbox!
 
+    --
+    -- Water line geometries
+    --
+    UNION
+
+    SELECT name,
+           NULL AS area,
+           waterway AS kind,
+           'openstreetmap.org' AS source,
+           way AS __geometry__,
+           mz_id AS __id__
+
+    FROM planet_osm_line
+
+    WHERE
+        waterway IN ('canal', 'river')
```

Comment:
Also, are there ever any waterways that we'd want to show that don't have a name set?

If we only want to show those that have names, we can add a filter where `name IS NOT NULL` and include that in the partial index. This would most likely help with index size and performance.